onnxruntime/test/providers/qnn/argmaxmin_op_test.cc

```diff
@@ -90,10 +98,12 @@ static void RunQDQArgMxxOpTest(const std::string& op_type, TestInputDef<float> i
   provider_options["backend_path"] = "libQnnHtp.so";
 #endif
 
-  RunQnnModelTest(BuildQDQArgMxxTestCase(op_type, input_def, attrs),
-                  provider_options,
-                  opset,
-                  expected_ep_assignment);
+  TestQDQModelAccuracy(BuildArgMxxTestCase(op_type, input_def, attrs),            // baseline float32 model
+                       BuildQDQArgMxxTestCase<QType>(op_type, input_def, attrs),  // QDQ model
+                       provider_options,
+                       opset,
+                       expected_ep_assignment,
+                       1e-5f);
```

Comment:
Some of my thoughts: Here we need to build a fp32 model, then a QDQ model after we got the output from fp32 model. Is it possible that we only need to build a fp32 model? For example, pass a flag with builder to the BuildArgMxxTestCase to know whether it's fp32 model or not. 
std::function<void(ModelTestBuilder& builder, std::vector<QuantParams<QuantType>>& output_qparams, bool fp32_model, std::string op_type)>
We can insert the QDQ nodes based on op_type.

Reply:
Do you mean that instead of passing two model building functions, we just use a single function can build both QDQ and float32 models?

I think that's a good idea because it reduces the number of parameters to `TestQDQModelAccuracy` and it is easier to ensure that both of the models have the same input/output types. I can give it a try.

Reply:
As mentioned in our separate conversation, I'll explore implementing this in the next PR that adds more unit tests.